public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -193,22 +191,22 @@ table(width="100%")
         ### bind to the `click` event
       +makeExample('cb-a1-a2-quick-reference/ts/app/app.component.html', 'event-binding')(format="." )
       :marked
-        The Angular&nbsp;1 event-based directives do not exist in Angular&nbsp;2.
-        Rather, we define one-way binding from the template view to the component using **event binding**.
+        Angular&nbsp;1 event-based directives do not exist in Angular&nbsp;2.
+        Rather, define one-way binding from the template view to the component using **event binding**.
 
-        For event binding, we define the name of the target event within parenthesis and
-        specify a template statement in quotes to the right of the equals. Angular&nbsp;2 then
+        For event binding, define the name of the target event within parenthesis and
+        specify a template statement, in quotes, to the right of the equals. Angular&nbsp;2 then
         sets up an event handler for the target event. When the event is raised, the handler
         executes the template statement.
 
-        In the first example, when the button is clicked, the `toggleImage()` method in the associated component is executed.
+        In the first example, when a user clicks the button, the `toggleImage()` method in the associated component is executed.
 
         The second example demonstrates passing in the `$event` object, which provides details about the event
         to the component.
 
         For a list of DOM events, see: https://developer.mozilla.org/en-US/docs/Web/Events.
 
-        For more information see [Template Syntax](../guide/template-syntax.html#event-binding).
+        For more information, see [Template Syntax](../guide/template-syntax.html#event-binding).
```

Comment:
Another possibility (especially in cases where it sounds like you're repeating the section name too much) is to not explicitly name the section, but to otherwise indicate that you're not taking them to the top of Template Syntax.